Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1047,7 +1047,7 @@ private void SetStaticProperties()
         private async void PluginInstalled(object sender, GenericEventArgs<PackageVersionInfo> args)
         {
             string dir = Path.Combine(ApplicationPaths.PluginsPath, Path.GetFileNameWithoutExtension(args.Argument.targetFilename));
-            var types = Directory.EnumerateFiles(dir, "*.dll", SearchOption.TopDirectoryOnly)
+            var types = Directory.EnumerateFiles(dir, "*.dll", SearchOption.AllDirectories)
```

Comment:
I don't think this change is necessary

Reply:
Yes, this one is iffy. With or without this change no plugins seemed to load without a restart, so zi couldn't verify how required it was. I changed it so they're both the same.